docs/UseCases.md

```diff
@@ -0,0 +1,64 @@
+# Use Cases
+
+## DAOs migrating to own chain
```

Comment:
Maybe worth also breaking out a new use case that's fully replicated? As a use case, one may want a "full consumer" chain, that uses 100% of the native chain voting power for operation.

Reply:
I will mention that. As I have designed, they always need a native token, so many 99.9% from the provider. 

We will never do proper replicated security as (1) there are no constraints to match validator sets and (2) it is opt-in from the provider chain, so it will generally be a percentage of that. 

I can mention one point closer to replicated security though

Reply:
I added a new section on "Almost Replicated Security" happy for feedback (even after merge)